build/BUILD.mariadb

```diff
@@ -1,3 +1,15 @@
+load("@bazel_skylib//lib:selects.bzl", "selects")
+
+config_setting(
+    name = "i686",
+    values = {"cpu": "i686"},
+)
+
+selects.config_setting_group(
```

Comment:
remove, just use `":i686": """cat > $@ <<"EOF"` at line 249.
```suggestion

```

Reply:
I prefer to keep it like that because I expect other 32 bits flavors to be requested in the future so it'll be easier to add them this way.

Reply:
what do you mean `other 32 bits`? can you give some examples? 
don't see any benefit from it, it's not backwards compatible.

Reply:
"i386", "i486", "i586", "i686", "i786", "x86" are all potential 32 bits CPUs.